### ISSUE TITLE
Update dependecies

### ISSUE THREAD

**Issue by shahryarjb** [kipcole9/url#3]
Hello, Dear @kipcole9 .
Is this library still supported? Because its dependencies have not been updated in the mix file for a long time.
I wanted to send a pull request by updating these dependencies before saying that maybe you have another strategy in mind

```
Dependencies (5)
ex_cldr
ex_phon_number
gettext
jason
nimble_parsec
```

And there are some `deprecated` things like config file in new versions of elixir

```
Erlang/OTP 26 [erts-14.0.2] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [jit:ns]

Elixir 1.15.1 (compiled with Erlang/OTP 26)
```

Thank you in advance

**Comment by kipcole9** [comment]
Definitely still supported.  I'll make sure there are no warning on updates dependencies this weekend - I appreciate you opening the issue.

**Comment by kipcole9** [comment]
I've updated dependencies.  I've fixed the `import Mix.Config` warnings and moved to `import Config` - but this only affects the development environment, not library consumers.

I don't see any library issues with OTP 26 or Elixir 1.15. 

I do see that both `sweet_xml` and `ex_lib_phonenumber` do have warnings about `Logger.warn/1` so I will send a PR to both of those libraries on the weekend.

Normally I would consider dependency updates as a reason to publish a new version - but it is valuable for testing of course.

Do you see any other issues?

**Comment by shahryarjb** [comment]
Thank you sir, I run mix test in my terminal and all the test are passed, if I find another issue or problems I create new issue for that, I will use inside my [macro](https://github.com/mishka-group/mishka_developer_tools/blob/master/lib/macros/guarded_struct.ex) 
if you release a version it helps me to do my staff 🥹💐
Thank you in advance

**Comment by kipcole9** [comment]
Closing for now. I have published [URL version 1.5.0](https://hex.pm/packages/url/1.5.0) with the following changelog entry:

### Bug Fixes

* Change to `import Config`, not `import Mix.Config`

* Make `Jason` dependency optional.

Note that this release, and later releases, are supported on Elixir 1.11 and later only.
